File: MtxVec.Delphi/Basic3.pas
unit Basic3;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  StdCtrls, ComCtrls;

type
  TBasicForm3 = class(TForm)
    RichEdit1: TRichEdit;
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public                                                             
    { Public declarations }
  end;

var
  BasicForm3: TBasicForm3;

implementation

{$IFDEF CLR}
{$R *.NFM}
{$ELSE}
{$R *.DFM}
{$ENDIF}

procedure TBasicForm3.FormCreate(Sender: TObject);
begin
     RichEdit1.Lines.Clear;
     RichEdit1.DefAttributes.Size := 10;
     RichEdit1.DefAttributes.Style := [];
end;

end.

File: Signal.CPP/Basic3.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "Basic3.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TBasicForm3 *BasicForm3;
//---------------------------------------------------------------------------
__fastcall TBasicForm3::TBasicForm3(TComponent* Owner)
  : TForm(Owner)
{
  RichEdit1->Lines->Clear();
}
//---------------------------------------------------------------------------


